### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by genesis331**
- [/] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/master/CONTRIBUTING.md)?
- [/] Have you explained what your changes do, and why they add value to the Guides?

Just a typo.

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

### CODE REVIEW

File: _articles/zh-cn/how-to-contribute.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ related:
   </p>
 </aside>
 
-即使你是以为开发者，非代码的贡献对于项目来说也是举足轻重的，尤其是对于社区的其他成员来说。用心维系这些关系能够让你有工作到项目其它部分的机会。
+即使你是一位开发者，非代码的贡献对于项目来说也是举足轻重的，尤其是对于社区的其他成员来说。用心维系这些关系能够让你有工作到项目其它部分的机会。
```
Comment (mlinksva) [inline _articles/zh-cn/how-to-contribute.md]: Autotranslation old:

> Even if you think of a developer, the contribution of non-code is very important to the project, especially for other members of the community. Carefully maintaining these relationships can give you the opportunity to work on other parts of the project.

New:

> Even if you are a developer, non-code contributions are important to the project, especially for other members of the community. Carefully maintaining these relationships will give you the opportunity to work on other parts of the project.

Original:

> Even if you’re a software developer, working on a documentation project can help you get started in open source. It’s often less intimidating to work on projects that don’t involve code, and the process of collaboration will build your confidence and experience.

So this PR looks good to me. Can any Chinese speaker give a second opinion?

Comment (genesis331) [reply _articles/zh-cn/how-to-contribute.md]: Hi @mlinksva , yes it looks good in the translator.
However, if you don't translate the whole sentence. 

The result shows this (from Google Translate): 
"即使你是以为开发者" (Original)
> Even if you think of a developer

"即使你是一位开发者" (New)
> Even if you are a developer

But, I hope a Chinese speaker can review this too.

Comment (mlinksva) [reply _articles/zh-cn/how-to-contribute.md]: cc @lijiangsheng1 who I believe did the initial translation in #443 to doublecheck this fix.

Comment (mlinksva) [reply _articles/zh-cn/how-to-contribute.md]: I'm going to go ahead and merge as this looks good to me. If another Chinese speaker sees this and can further improve, please open another issue or PR! 🙇

Comment (genesis331) [reply _articles/zh-cn/how-to-contribute.md]: @mlinksva Thanks!

**Pull request merged**